ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdapter.java

```diff
@@ -274,20 +212,64 @@ public void showInterstitial() {
       return;
     }
 
-    if (!UnityAds.isReady(mPlacementId)) {
+    if (mPlacementId == null) {
       Log.w(TAG,
           "Unity Ads failed to show interstitial ad for placement ID '" + mPlacementId +
               "'. Placement is not ready.");
       mMediationInterstitialListener.onAdClosed(UnityAdapter.this);
-      return;
+    } else {
+      mPlacementsInUse.remove(mPlacementId);
```

Comment:
mPlacementsInUse does not line up with iOS implementation or Mopub (android and iOS) implementation

Reply:
This was already in there though and might be better addressed in our followup refactor: https://github.com/googleads/googleads-mobile-android-mediation/blob/master/ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdapter.java#L88